src/main/java/org/terasology/changingBlocks/conditional/components/ChangeBlockBlockDirectedComponent.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.changingBlocks.conditional.components;
+
+import org.terasology.changingBlocks.conditional.BlockCondition;
+
+import java.util.List;
+
+public class ChangeBlockBlockDirectedComponent implements ConditionalBlockChangeComponent {
```

Comment:
This name reads funny with the double BlockBlock in the middle, and I don't think it's intuitive what this is about... Why do we need to treat them separately? Would a combined `BlockChangingComponent` work? 

```java
public class BlockChangeComponent implements ConditionalBlockChangeComponent {
	public List<BlockCondition.BlockDirected> directedConditions;
	public List<BlockCondition.BlockNearby> nearbyConditions;
    ...
}
```

I'm not really fond of the number of component classes we need to model the different changes...